core/app/api/customer/group/route.ts

```diff
@@ -0,0 +1,9 @@
+import { NextResponse } from 'next/server';
+
+import { getCustomer } from '~/client/queries/get-customer';
+
+export async function GET() {
+  const { data } = await getCustomer();
+
+  return NextResponse.json(data);
```

Comment:
Is it okay to expose the customer like this in an unauthenticated endpoint? We should probably expose only what's needed to implement the customer group functionality to avoid leaking sensitive user info